Sources/FrontEnd/TypeChecking/ConstraintGenerator.swift

```diff
@@ -14,8 +14,8 @@ struct ConstraintGenerator {
     /// The set of type constraints being generated.
     let constraints: [Constraint]
 
-    /// The diagnostics of the errors the generator encountered.
-    let diagnostics: [Diagnostic]
+    /// Indicates that the solver encountered one or more errors.
+    let didFoundError: Bool
```

Comment:
```suggestion
    let foundError: Bool
```
Although the above name is slightly ambiguous; it could be the error that was found.

```suggestion
    let errorFound: Bool
```

doesn't have that problem.

This is the old Cocoa style which seems pointless to me.
```suggestion
    let didFindError: Bool
```

Reply:
Sorry, I missed that there were two occurrences of properties with this name. I replaced (now instances) by:
```swift
/// True iff a constraint could not be solved.
let foundConflict: Bool
```